### ISSUE TITLE
embedding ethercalc within another page

### ISSUE THREAD

**Issue by rimmartin** [audreyt/ethercalc#308]
Maybe this has answered somewhere yet with the large volume of info about ethercalc I haven't found it.

Is this possible?  Can ethercalc be embedded within another page? I want to put a jstree to the left and also support multiviews of the same data on the same page.

Any examples?

**Comment by audreyt** [comment]
Yes certainly, simply put them into an iframe will do.

**Comment by rimmartin** [comment]
Hi,  worded this wrong.  I was meaning the spreadsheet when I wrote ethercalc

When I started the server with
```
./bin/ethercalc
```
and start my  dependent pages with 
```
<div id="table-spreadsheet"><iframe src="http://hostname:8000/"/></div>
```
The first iframed page is like
https://ethercalc.org/ which requires clicking the 'Create Spreadsheet' button to then get to the spreadsheet while I'm trying to have the spreadsheet itself show immediately and have data from my hdf5 source populate and readied for the user

Maybe I need to add some client scripting to auto-press the button for me?  And then begin scripting the data flow to the sheet?
